### ISSUE TITLE
Fails silently on telegram-desktop on macOS

### ISSUE THREAD

**Issue by jokoso** [nathanielevan/gruvbox-material-telegram#1]
Hi! Thanks for making this.

I'm having trouble applying this theme. When I attempt to apply it, nothing
happens; no theme is added and there is no error shown. This is the case for
both the link in the README and the build artifact.

Have anyone else run into this problem?

App: `telegram-desktop` version `3.7.3` installed using Homebrew

**Comment by nathanielevan** [comment]
I can confirm this happens to me as well, and I think I found a solution. Try zipping the files manually, and then change the extension of the resulting .zip file to .tdesktop-theme; does that solve the issue? Let me know if it does so I can push the change, thanks!

As for the link, yikes, I just realised that the link should've been https://t.me/addtheme/gruvmaterialdark instead of gruvboxmaterialdark. Will update accordingly, thanks for the report!

**Comment by nathanielevan** [comment]
I can confirm this happens to me as well, and I think I found out why. The zip command that I use in the Makefile retains the src folder in the archive and puts the theme files in that folder, resulting in Telegram not being able to find them.

As for the link, yikes, I just realised that the link should've been https://t.me/addtheme/gruvmaterialdark instead of gruvboxmaterialdark. Will update accordingly, thanks for the report!
